src/aws_s3_auth_conn.tsx

```diff
@@ -59,11 +86,23 @@ export class MyS3Auth {
   // Check if the user's keys are valid for specified bucket
   // Save the S3 object made upon validation
   // Code after this function call will likely execute before this function finishes
-  checkAndDisplayValidUser(setViewState : (args : string) => any, args : string, setS3Obj : (s3Obj : MyS3Auth) => any, s3Obj : MyS3Auth ) {
+  checkAndDisplayValidBucket(setViewState : (args : string) => any, args : string, setS3Obj : (s3Obj : MyS3Auth) => any, s3Obj : MyS3Auth ) {
```

Comment:
function name probably not super representative of functionality

Reply:
Said to leave the stale function